src/guides/v2.4/graphql/interfaces/configurable-product.md

```diff
@@ -738,6 +758,429 @@ The following `products` query returns `ConfigurableProduct` information about t
+              "label": "",
+              "position": 2
+            },
+            {
+              "url": "http://<host>/media/catalog/product/cache/816455256c48217ab8c5c822a6039d1a/m/s/ms10-blue_back_1.jpg",
+              "label": "",
+              "position": 3
+            }
+          ],
+          "configurable_options": [
+            {
+              "uid": "Y29uZmlndXJhYmxlLzYwMi85Mw==",
+              "attribute_code": "color",
+              "values": [
+                {
+                  "label": "Black",
+                  "uid": "Y29uZmlndXJhYmxlLzkzLzQ5"
+                },
+                {
+                  "label": "Blue",
+                  "uid": "Y29uZmlndXJhYmxlLzkzLzUw"
+                },
+                {
+                  "label": "Red",
+                  "uid": "Y29uZmlndXJhYmxlLzkzLzU4"
+                }
+              ]
+            },
+            {
+              "uid": "Y29uZmlndXJhYmxlLzYwMi8xNjA=",
+              "attribute_code": "size",
+              "values": [
+                {
+                  "label": "XS",
+                  "uid": "Y29uZmlndXJhYmxlLzE2MC8xNjY="
+                },
+                {
+                  "label": "S",
+                  "uid": "Y29uZmlndXJhYmxlLzE2MC8xNjc="
+                },
+                {
+                  "label": "M",
+                  "uid": "Y29uZmlndXJhYmxlLzE2MC8xNjg="
+                },
+                {
+                  "label": "L",
+                  "uid": "Y29uZmlndXJhYmxlLzE2MC8xNjk="
+                },
+                {
+                  "label": "XL",
+                  "uid": "Y29uZmlndXJhYmxlLzE2MC8xNzA="
+                }
+              ]
+            }
+          ],
+          "configurable_product_options_selection": {
+            "options_available_for_selection": [
+              {
+                "attribute_code": "size",
+                "option_value_uids": [
+                  "Y29uZmlndXJhYmxlLzE2MC8xNjY=",
+                  "Y29uZmlndXJhYmxlLzE2MC8xNjc=",
+                  "Y29uZmlndXJhYmxlLzE2MC8xNjg=",
+                  "Y29uZmlndXJhYmxlLzE2MC8xNjk=",
+                  "Y29uZmlndXJhYmxlLzE2MC8xNzA="
+                ]
+              },
+              {
+                "attribute_code": "color",
+                "option_value_uids": [
+                  "Y29uZmlndXJhYmxlLzkzLzU4"
+                ]
+              }
+            ],
+            "media_gallery": [
+              {
+                "url": "http://<host>/media/catalog/product/cache/816455256c48217ab8c5c822a6039d1a/m/s/ms10-red_main_1.jpg",
+                "label": "",
+                "position": 1
+              }
+            ],
+            "variant": null
+          }
+        }
+      ]
+    }
+  }
+}
+```
+
+#### Step 3. Select the size
+
+In this example, the shopper has selected the Medium option for the size attribute. The query adds the corresponding `uid` to the `configurable_product_options_selection` filter.
```

Comment:
```suggestion
In this example, because the shopper has selected the Medium option for the size attribute, the query adds the corresponding `uid` to the `configurable_product_options_selection` filter.
```